### PR TITLE
Create offboarding.md

### PR DESCRIPTION
**Pull request by tiffanychu90**
# Description

_Describe your changes and why you're making them. Please include the context, motivation, and relevant dependencies._

We've cycled through several offboarding of contractors and consultants and we should keep track of several updates we need to make to our repos.

Resolves #\[issue\]

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## How has this been tested?

_Include commands/logs/screenshots as relevant._

## Post-merge follow-ups

_Document any actions that must be taken post-merge to deploy or otherwise implement the changes in this PR (for example, running a full refresh of some incremental model in dbt). If these actions will take more than a few hours after the merge or if they will be completed by someone other than the PR author, please create a dedicated follow-up issue and link it here to track resolution._

- [ ] No action required
- [x] Actions required (specified below)
  Do a pass and remove previous contractors and consultants from admin privileges and update references in `data-infra` that send notifications to closed email accounts.

### CODE REVIEW

**Comment by tiffanychu90** [comment]
Google IAM
**Jarvus / Slalom**
- [ ] James Lott `james.l@jarv.us` - all permissions not used in last 90 days 
- [ ] Soren Spicknall `soren.s@jarv.us`
- [ ] Mjumbe Poe  `mjumbe.p@jarv.us`- most permissions are in excess
- [ ] Kim Engie `kim.engie@slalom.com`

**Caltrans**
- [ ] Julian Hernandez  `julian@calitp.org`
- [ ] Samantha Lewis `samantha@calitp.org`

GitHub
- [ ] references to Andrew or Soren's emails (`andrew.v@jarv.us` or `soren.s@jarv.us`)
- [ ] Kubernetes - Michael Chow still referenced (`machow`)

File: .github/ISSUE_TEMPLATE/offboarding.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - [ ] Team: warehouse-users and warehouse-contributors
+  - [ ] Update code references to designated hand-off staff or hello@calitp.org
+  - [ ] Remove administrative privileges / pull request approvals
+
+- [ ] JupyterLab
+
+- [ ] Google IAM (Cloud and BQ)
+
+- [ ] Metabase
+
```
Comment (evansiroky) [inline .github/ISSUE_TEMPLATE/offboarding.md:34]: Can you add Airtable in here?

**Review by evansiroky — CHANGES_REQUESTED** [review]
See comments

**Comment by tiffanychu90** [comment]
> See comments

Added Airtable, @evansiroky. I'll open up another issue to follow-up with actual offboarding with Google and (hopefully) Github permissions with the template.

**Review by evansiroky — APPROVED** [review]

**Pull request merged**